src/dotnet/AutoRest/Plugins/AzureValidator.cs

```diff
@@ -90,12 +116,6 @@ private void LogValidationMessage(ValidationMessage validationMessage)
                 throw new Exception("Invalid Input for openapi-type: " + docTypeInput + ". Valid values are 'arm', 'data-plane' or 'default'.");
             }
 
-            ServiceDefinitionDocumentState docState;
```

Comment:
Why are we removing this validation?

Reply:
Not removed, just moved

Reply:
Moved to...? 
In `ProcessInternal` you're not throwing an exception for invalid inputs.

Reply:
Line 76

Reply:
Ahh darn! that should do